ci-viewer.members.txt

```diff
@@ -7,3 +7,5 @@ holden.karau@gmail.com
 iman.tabrizian@gmail.com
 nrchakradhar379@gmail.com
 redhat-team@kubeflow.org
+dsun20@bloomberg.net
```

Comment:
Alphabetical order please.

Reply:
Updated:)

Reply:
@jlewi let me know if this pr looks good to you :)